script.myepisodes/resources/language/English/strings.po

```diff
@@ -0,0 +1,86 @@
+# XBMC Media Center language file
```

Comment:
change the language layout folder please: `/resources/language/resource.language.en_gb/strings.po`. New defaults since krypton